go/vt/binlog/binlogplayer/binlog_player.go

```diff
@@ -600,31 +601,36 @@ func ReadVRSettings(dbClient DBClient, uid uint32) (VRSettings, error) {
 	if len(qr.Rows) != 1 {
 		return VRSettings{}, fmt.Errorf("checkpoint information not available in db for %v", uid)
 	}
-	vrRow := qr.Rows[0]
+	vrRow := qr.Named().Row()
```

Comment:
Thank you for refactoring this usage as we go!

Reply:
This again was stolen from #9742 authored by @rohit-nayak-ps